extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -28,7 +28,13 @@ public function testRenderInfobox( $input, $expectedOutput, $description ) {
 		$actualDOM->loadXML($actualOutput);
 		$expectedDOM->loadXML($expectedOutput);
 
-		$this->assertEquals( $expectedDOM->saveXML(), $actualDOM->saveXML(), $description );
+		// lets sanitize expected and actual HTML to not deal with whitespaces
```

Comment:
This looks scary. A link to source explaining why this is needed (in the light of DOMDocument attributes set above) should be added here.